rfc/draft-oberstet-hybi-tavendo-wamp-00.md

```diff
@@ -41,91 +41,84 @@ This document defines the Web Application Messaging Protocol (WAMP). WAMP is a r
 
-Routed Remote Procedure Calls transfer the decoupling of the Publish & Subscribe pattern to Remote Procedure Calls. A caller is no longer required to have knowledge of the Callee, it merely needs to know the identifier for the procedure it wants to call. There is also no longer a need for a direct connection between the caller and the callee, since all traffic is routed. This enables calling procedures in components which are not reachable externally, e.g. on a NATted connection, but which can establish an outgoing connection to the WAMP router.
+The decoupling in routed RPCs arises from the fact that the Caller is no longer required to have knowledge of the Callee; it merely needs to know the identifier of the procedure it wants to call. There is also no longer a need for a direct connection between the caller and the callee, since all traffic is routed. This enables the calling of procedures in components which are not reachable externally (e.g. on a NATted connection) but which can establish an outgoing connection to the WAMP router.
 
-Combining these two patterns into a single protocol allows a single protocol to be used for the entire messaging requirements of an application, reducing complexity in the technology stack and networking overheads.
+Combining these two patterns into a single protocol allows it to be used for the entire messaging requirements of an application, thus reducing technology stack complexity, as well as networking overheads.
 
-While WAMP was originally specified to run over WebSocket, it can run over any transport which is message-based, ordered, reliable and bi-directional.
+While WAMP was originally specified to run over WebSocket, it can run over any transport which is message-oriented, ordered, reliable, and bi-directional.
 
 ## Protocol Overview
 
 _This section is non-normative._
 
-For each of the two messaging patterns, three roles are defined:
+The PubSub messaging pattern defines three roles: _Subscribers_ and _Publishers_, which communicate via a _Broker_.
 
-For PubSub, there are Subscribers and Publishers, which are connected through a Broker.
+The routed RPC messaging pattern also defines three roles: _Callers_ and _Callees_, which communicate via a _Dealer_.
 
-For routed Remote Procedure Calls there are Callers and Callees, which are connected through a Dealer.
+WAMP Connections are established by Clients to a Router. Connections can use any transport that is message-based, ordered, reliable and bi-directional, with WebSocket as the default transport.
```

Comment:
This sentence shouldn't be here (it's already on lilne 74). I'll remove it in the next commit.